jose-jwt/crypto/AesGcmNetCore.cs

```diff
@@ -37,7 +45,14 @@ public static byte[][] Encrypt(byte[] key, byte[] iv, byte[] aad, byte[] plainTe
         /// <exception cref="CryptographicException">if decryption failed by any reason</exception>
         public static byte[] Decrypt(byte[] key, byte[] iv, byte[] aad, byte[] cipherText, byte[] authTag)
         {
+#if NET
```

Comment:
i guess we can just collapse both branches together now? No need to use .NET 8.0 specific AesGcm() constructor as you check auth tag length explicitly now. And you can leverage `Ensure.BitSize(..)` helper right at the start of the method.

And after just clean up all `NET` targeting, cause no need to and PR will become nice & small, just of couple files :)